docs/reference/cpp_language_extensions.rst

```diff
@@ -292,8 +292,7 @@ dimensions to 1.
 Memory fence instructions
 ====================================================
 
-HIP supports ``__threadfence()`` and ``__threadfence_block()``. If you're using ``threadfence_system()`` in
-the HIP-Clang path, you can use the following workaround:
+HIP supports ``__threadfence()`` and ``__threadfence_block()``. If you're using ``threadfence_system()`` in the HIP-Clang path, you can use the following workaround:
```

Comment:
`__threadfence_system()` is available in HIP.  No workaround is needed.

Reply:
Fixed.